src/test/java/org/springframework/cloud/deployer/scheduler/spi/kubernetes/EntryPointStyleTests.java

```diff
@@ -0,0 +1,30 @@
+package org.springframework.cloud.deployer.scheduler.spi.kubernetes;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for {@link EntryPointStyle}
+ *
+ * @author Chris Schaefer
+ */
+public class EntryPointStyleTests {
```

Comment:
this might shuffle elsewhere if we go with a single `EntryPointStyle` enum rather than dups